test/functional/test-performance.js

```diff
@@ -480,12 +481,13 @@ describe('performance', () => {
         whenReadyToRetrieveResourcesResolve();
         whenViewportLayoutCompleteResolve();
         return perf.whenViewportLayoutComplete_().then(() => {
-          expect(tickSpy.callCount).to.equal(2);
-          expect(tickSpy.firstCall.args[0]).to.equal('_pc');
-          expect(tickSpy.secondCall.args[0]).to.equal('pc');
-          expect(tickSpy.secondCall.args[1]).to.equal('_pc');
-          expect(Number(tickSpy.firstCall.args[2])).to.equal(perf.initTime_);
-          expect(Number(tickSpy.secondCall.args[2])).to.equal(
+          expect(tickSpy.callCount).to.equal(3);
+          expect(tickSpy.firstCall.args[0]).to.equal('ol');
```

Comment:
`ol` tick happens from the constructor. Does that answer your question? I don't know much about performance so my answers might be stupid.